docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cmake \
     ../src
 RUN make VERBOSE=1 -j2
 RUN ccache -s
-RUN ctest -E "(walker:SkewNormalPDF_pe8|walker:DiagOrnsteinUhlenbeckPDF_pe8)"
+RUN ctest -E "_pe[45678]"
```

Comment:
BTW, the `_pe*` tests attempt to brute-force test asynchronous logic: the runtime system will schedule some calls differently depending on the load of the machine the tests are running on. So if they fail sometimes and apparently in a random fashion or some tests stall forever, that's an indication of an asynchronous logic bug, which can be pretty hard to debug. Bottom line: it is nice to include as many of such tests.